src/store/modules/job/getters.ts

```diff
@@ -39,6 +40,30 @@ const getters: GetterTree <JobState, RootState> = {
     getJobs: (state) => {
       return state.cached;
     },
+    getIncludedTagsAndOperator: (state) => (id: string): any => {
+      const thresholdRule = state.thresholdRules[id];
+      if (!thresholdRule) return "";
+      const tagsIncluded = thresholdRule.json.filter.find((filter: any) => filter.startsWith("tags:"))
+      if (!tagsIncluded) return ""
+      let tags = parser.parseBooleanQuery(tagsIncluded.substring(tagsIncluded.indexOf(":") + 1).trim());
+      const operator = tagsIncluded.indexOf('AND') ? 'AND' : 'OR'
+      tags = tags.map((x: any) => {
+        return JSON.parse(x[0])
+      });
+      return { tags, operator }
+    },
+    getExcludedTagsAndOperator: (state) => (id: string): any => {
+      const thresholdRule = state.thresholdRules[id];
+      if (!thresholdRule) return "";
+      const tagsIncluded = thresholdRule.json.filter.find((filter: any) => filter.startsWith("-tags:"))
+      if (!tagsIncluded) return ""
+      let tags = parser.parseBooleanQuery(tagsIncluded.substring(tagsIncluded.indexOf(":") + 1).trim());
+      const operator = tagsIncluded.indexOf('AND') ? 'AND' : 'OR'
+      tags = tags.map((x: any) => {
+        return JSON.parse(x[0])
```

Comment:
We don't need return here, we can directly write JSON.parse(x[0])

Reply:
Made the requested change.